Form/Extension/DescriptionFormTypeExtension.php

```diff
@@ -50,6 +59,6 @@ public function setDefaultOptions(OptionsResolverInterface $resolver)
      */
     public function getExtendedType()
     {
-        return 'form';
+        return LegacyFormHelper::getType('Symfony\Component\Form\Extension\Core\Type\FormType');
```

Comment:
I took it from original repository, i would like not to change to our approach to give a easiest possiblillity to upgrade later.